.github/workflows/release.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
           profile: minimal
           toolchain: nightly
 
+      - name: Install Solana
```

Comment:
do we need Solana here? afaik this workflow only uses anchor to generate the idls via cargo